az-crypto/src/main/java/azkaban/crypto/Crypto.java

```diff
@@ -38,11 +39,11 @@ public Crypto() {
   }
 
   public static String encode(final String s) {
-    return Base64.getEncoder().encodeToString(s.getBytes());
+    return Base64.getEncoder().encodeToString(s.getBytes(Charset.defaultCharset()));
```

Comment:
Should we use UTF-8 instead of default character setting?

Reply:
thanks. updated.